-1 
Tested at: 64fa8130e807aea6ee672d6e7213aad120dfd49c
 When I ran the RelVals I found an error in the following worklfows: 
 134.911 step2
<pre>runTheMatrix-results/134.911_RunSinglePh2015D+RunSinglePh2015D+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT/step2_RunSinglePh2015D+RunSinglePh2015D+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT.log

----- Begin Fatal Exception 04-Dec-2015 08:08:23 CET-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing input source of type PoolSource

   [2] Calling RootFileSequenceBase::initTheFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/data/Run2015D/SinglePhoton/RAW/v1/000/256/677/00000/1ADDE997-605D-E511-944F-02163E01424B.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms.cern.ch//eos/cms/store/data/Run2015D/SinglePhoton/RAW/v1/000/256/677/00000/1ADDE997-605D-E511-944F-02163E01424B.root', flags=0x10, permissions=0660) => error '[ERROR] Operation expired' (errno=0, code=206). No additional data servers were found.

      [b] Last URL tried: root://eoscms.cern.ch:1094//eos/cms/store/data/Run2015D/SinglePhoton/RAW/v1/000/256/677/00000/1ADDE997-605D-E511-944F-02163E01424B.root?tried=

      [c] Problematic data server: eoscms.cern.ch:1094

      [d] Disabled source: eoscms.cern.ch:1094

      [e] Input file root://eoscms.cern.ch//eos/cms/store/data/Run2015D/SinglePhoton/RAW/v1/000/256/677/00000/1ADDE997-605D-E511-944F-02163E01424B.root could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/data/Run2015D/SinglePhoton/RAW/v1/000/256/677/00000/1ADDE997-605D-E511-944F-02163E01424B.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/data/Run2015D/SinglePhoton/RAW/v1/000/256/677/00000/1ADDE997-605D-E511-944F-02163E01424B.root', flags=0x10, permissions=0660) => error '[FATAL] Redirect limit has been reached' (errno=0, code=306). No additional data servers were found.

      [g] Last URL tried: root://cmsxrootd-site2.fnal.gov:1093//store/data/Run2015D/SinglePhoton/RAW/v1/000/256/677/00000/1ADDE997-605D-E511-944F-02163E01424B.root?tried=+1213xrootd-redic.pi.infn.it,

      [h] Problematic data server: cmsxrootd-site2.fnal.gov:1093

      [i] Disabled source: cmsxrootd-site2.fnal.gov:1093

----- End Fatal Exception -------------------------------------------------

</pre>

 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-12664/10131/summary.html

--- a/Configuration/AlCa/python/autoCond.py
+++ b/Configuration/AlCa/python/autoCond.py
@@ -10,23 +10,25 @@ autoCond = {
     # GlobalTag for MC production (p-Pb collisions) with realistic alignment and calibrations for Run1
     'run1_mc_pa'        :   '76X_mcRun1_pA_v10',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Run2
-    'run2_design'       :   '76X_mcRun2_design_v12',
+    'run2_design'       :   '76X_mcRun2_design_v13',
     # GlobalTag for MC production with pessimistic alignment and calibrations for Run2
-    'run2_mc_50ns'      :   '76X_mcRun2_startup_v11',
+    'run2_mc_50ns'      :   '76X_mcRun2_startup_v12',
     #GlobalTag for MC production with optimistic alignment and calibrations for Run2
-    'run2_mc'           :   '76X_mcRun2_asymptotic_v12',
+    'run2_mc'           :   '76X_mcRun2_asymptotic_v13',
     # GlobalTag for MC production (Heavy Ions collisions) with optimistic alignment and calibrations for Run2
     'run2_mc_hi'        :   '76X_mcRun2_HeavyIon_v12',
     # GlobalTag for Run1 data reprocessing
-    'run1_data'         :   '76X_dataRun1_v11',
+    'run1_data'         :   '76X_dataRun1_v15',
     # GlobalTag for Run2 data reprocessing
-    'run2_data'         :   '76X_dataRun2_v11',
+    'run2_data'         :   '76X_dataRun2_v15',
     # GlobalTag for Run1 HLT: it points to the online GT
-    'run1_hlt'          :   '76X_dataRun1_HLT_frozen_v9',
+    'run1_hlt'          :   '76X_dataRun1_HLT_frozen_v10',
     # GlobalTag for Run2 HLT: it points to the online GT
-    'run2_hlt'          :   '76X_dataRun2_HLT_frozen_v9',
+    'run2_hlt'          :   '76X_dataRun2_HLT_frozen_v10',
+    # GlobalTag for Run2 HLT for HI: it points to the online GT
+    'run2_hlt_hi'       :   '76X_dataRun2_HLTHI_v1',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2017
-    'phase1_2017_design' :  '76X_upgrade2017_design_v7',
+    'phase1_2017_design' :  '76X_upgrade2017_design_v8',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2019
     'phase1_2019_design' :  'DES19_70_V2', # placeholder (GT not meant for standard RelVal) 
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase2

